### ISSUE TITLE
[JETTY] Threads Busy keep increasing and not closes

### ISSUE THREAD

**Issue by dannyeuu** [metabase/metabase#8202]
### Bugs
If you're filing an issue about a bug please include as much information
as you can including the following.

- Your browser and the version: Chrome 67.0.3396.62 (Official Build) (64-bit)
- Your operating system: Ubuntu 18.04
- Your databases: MySQL, Postgres, SQL Server
- Metabase version: v0.29.3
- Metabase hosting environment: Linux/Debian
- Metabase internal database: Postgres

After restarting the service on users starting using when the logs get
`Jetty threads: 8/256 (254 busy, 2 idle, 0 queued)` It down the service, and Nginx Reverse Proxy gets only `HTTP_CODE 499`. On slow query, why the thread does not close?

Looking at Metabase database, I found this, loging a lot of query running til 20 minutes, but this never goes to the front.
![screenshot from 2018-08-02 19-38-59](https://user-images.githubusercontent.com/6808348/43614955-e72da610-968b-11e8-98b9-ffe6a788a723.png)

**Comment by achesnais** [comment]
We've also been experiencing this. Every once in a while new queries in the app stop responding, which forces us to restart Metabase.

I've found that triggering an x-ray on a big table triggers this immediately. Happy to provide further info if it can help diagnose.

**Comment by dannyeuu** [comment]
@senior There is an explanation about this?

**Comment by joaoferrao** [comment]
@dannyeuu how did you manage to increase the number of Jetty threads from 50?

**Comment by dannyeuu** [comment]
This is my Systemctl service @joaoferrao 

``` bash
[Unit]

Description=Metabase Service
After=network.target syslog.target

[Service]
PIDFile=/run/metabase/pid
User=www-data
Group=www-data
Type=simple
SyslogIdentifier=metabase
RuntimeDirectory=metabase
WorkingDirectory=/home/admin/metabase
Environment="JAVA_TOOL_OPTIONS=-Xmx6144m"
Environment="MB_JETTY_MAXTHREADS=4096"    #  <------- HERE ARE INCREASED
Environment="MB_JETTY_HOST=127.0.0.1"
Environment="MB_EMOJI_IN_LOGS=false"
Environment="MB_DB_DBNAME=****"
Environment="MB_DB_HOST=****"
Environment="MB_DB_PASS=****"
Environment="MB_DB_PORT=5432"
Environment="MB_DB_TYPE=postgres"
Environment="MB_DB_USER=metadataUser"
ExecStart=/bin/bash -c '/usr/bin/java -jar /home/admin/metabase/metabase.jar 2>> /home/admin/metabase/logs/metabase.stderr 1>> /home/admin/metabase/logs/metabase.stdout'
ExecReload=/bin/kill -s HUP $MAINPID
ExecStop=/bin/kill -s TERM $MAINPID

[Install]
WantedBy=multi-user.target
```

**Comment by jornh** [comment]
With #8312 resolved in 0.30.2 I’m wondering if this one got squashed as well?

Please retest with 0.30.2 just out. Thanks! 🙂

@dannyeuu  I’m not sure if you’re running https, so unsure if this has the criteria mentioned in https://github.com/metabase/metabase/issues/8312#issuecomment-417326691

**Comment by dannyeuu** [comment]
Resoved by v0.30.3
